eq-author/src/enhancers/withValidationError.js

```diff
@@ -0,0 +1,53 @@
+import PropTypes from "prop-types";
+import React from "react";
+import { propType } from "graphql-anywhere";
+import pageFragment from "graphql/fragments/page.graphql";
+
+import VALIDATION_MESSAGES from "constants/validationMessages";
+
+const withValidationError = WrappedComponent => {
+  return class extends React.Component {
+    static propTypes = {
+      enableValidationMessage: PropTypes.bool,
+      page: propType(pageFragment).isRequired,
```

Comment:
Why is it only page?

Reply:
Could we have it take any object and read its validationErrors? Or do you think thats too complex for the moment?

Reply:
Changed to be generic object as per withEntityEditor